cmd/agent/subcommands/flare/command_test.go

```diff
@@ -80,12 +88,33 @@ func (c *commandTestSuite) getPprofTestServer() (tcpServer *httptest.Server, uni
 	return tcpServer, unixServer
 }
 
+func (c *commandTestSuite) restartSystemProbeTestServer() {
+	t := c.T()
+
+	if runtime.GOOS == "windows" {
```

Comment:
instead of runtime conditional, suggest move empty implementation to `command_other_test.go`, and then have windows implementation here.

Reply:
Agreed. Since the test had an existing runtime check convention, I tried to keep it that way, but generally it is better to split it out.